CHANGELOG.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+
```

Comment:
I missed that discussion. The latest news I know is a @blaix 's requirement to move version to a separate pull.